tools/Makefile.host

```diff
@@ -104,6 +104,15 @@ ifdef HOSTEXEEXT
 mkversion: mkversion$(HOSTEXEEXT)
 endif
 
+# mkinfo - Convert a .info file into a C info.h file
+
+mkinfo$(HOSTEXEEXT): mkinfo.c cfgdefine.c
```

Comment:
why not use the similar approach of https://github.com/apache/nuttx-apps/blob/master/tools/host_sysinfo.py to generate nuttx info?

Reply:
@xiaoxiang781216 I suggest keep it as C to avoid dependence on python, case nxdialog become something adopted broadly for all arches. If eventually we got a C compiler like TCC working on NuttX we could compile NuttX inside a NuttX board. But if python is a requirement, it could make things hard to do.